Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642869): Gdk-CRITICAL **: 18:14:28.842: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641622): Gdk-CRITICAL **: 18:14:37.073: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641773): Gdk-CRITICAL **: 18:14:39.271: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641465): Gdk-CRITICAL **: 18:14:41.075: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640880): Gdk-CRITICAL **: 18:14:41.665: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642788): Gdk-CRITICAL **: 18:14:41.786: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641138): Gdk-CRITICAL **: 18:14:42.801: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642574): Gdk-CRITICAL **: 18:14:42.825: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642706): Gdk-CRITICAL **: 18:14:42.871: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640999): Gdk-CRITICAL **: 18:14:43.069: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640886): Gdk-CRITICAL **: 18:14:43.193: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642764): Gdk-CRITICAL **: 18:14:43.237: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642411): Gdk-CRITICAL **: 18:14:43.683: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642319): Gdk-CRITICAL **: 18:14:43.720: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640909): Gdk-CRITICAL **: 18:14:43.869: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641664): Gdk-CRITICAL **: 18:14:43.986: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641533): Gdk-CRITICAL **: 18:14:44.085: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642556): Gdk-CRITICAL **: 18:14:44.215: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1641475): Gdk-CRITICAL **: 18:14:44.228: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642806): Gdk-CRITICAL **: 18:14:44.262: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641515): Gdk-CRITICAL **: 18:14:44.431: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642820): Gdk-CRITICAL **: 18:14:44.557: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641882): Gdk-CRITICAL **: 18:14:44.507: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642201): Gdk-CRITICAL **: 18:14:44.572: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642122): Gdk-CRITICAL **: 18:14:45.653: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641440): Gdk-CRITICAL **: 18:14:45.944: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641032): Gdk-CRITICAL **: 18:14:46.176: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641852): Gdk-CRITICAL **: 18:14:46.245: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642154): Gdk-CRITICAL **: 18:14:46.705: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642778): Gdk-CRITICAL **: 18:14:47.178: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640933): Gdk-CRITICAL **: 18:14:47.195: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641822): Gdk-CRITICAL **: 18:14:47.259: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641098): Gdk-CRITICAL **: 18:14:47.402: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642546): Gdk-CRITICAL **: 18:14:47.548: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642257): Gdk-CRITICAL **: 18:14:47.529: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642311): Gdk-CRITICAL **: 18:14:48.784: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642838): Gdk-CRITICAL **: 18:14:48.900: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641589): Gdk-CRITICAL **: 18:14:49.059: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641313): Gdk-CRITICAL **: 18:14:49.238: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640911): Gdk-CRITICAL **: 18:14:49.321: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640913): Gdk-CRITICAL **: 18:14:50.964: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641062): Gdk-CRITICAL **: 18:14:51.021: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642494): Gdk-CRITICAL **: 18:14:51.095: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641388): Gdk-CRITICAL **: 18:14:51.134: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641760): Gdk-CRITICAL **: 18:14:51.511: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642797): Gdk-CRITICAL **: 18:14:51.847: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642427): Gdk-CRITICAL **: 18:14:51.983: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641131): Gdk-CRITICAL **: 18:14:52.226: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641626): Gdk-CRITICAL **: 18:14:52.667: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640905): Gdk-CRITICAL **: 18:14:53.305: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640926): Gdk-CRITICAL **: 18:14:54.397: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641644): Gdk-CRITICAL **: 18:14:54.475: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640941): Gdk-CRITICAL **: 18:14:54.641: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641719): Gdk-CRITICAL **: 18:14:55.514: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640979): Gdk-CRITICAL **: 18:14:55.678: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641682): Gdk-CRITICAL **: 18:14:55.719: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641076): Gdk-CRITICAL **: 18:14:55.740: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641609): Gdk-CRITICAL **: 18:14:55.957: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641560): Gdk-CRITICAL **: 18:14:56.031: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642073): Gdk-CRITICAL **: 18:14:56.086: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642751): Gdk-CRITICAL **: 18:14:56.339: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642456): Gdk-CRITICAL **: 18:14:56.617: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642785): Gdk-CRITICAL **: 18:14:56.786: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642742): Gdk-CRITICAL **: 18:14:56.913: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642718): Gdk-CRITICAL **: 18:14:57.388: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641840): Gdk-CRITICAL **: 18:14:59.388: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641702): Gdk-CRITICAL **: 18:14:59.441: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641011): Gdk-CRITICAL **: 18:14:59.463: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642529): Gdk-CRITICAL **: 18:15:00.302: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642337): Gdk-CRITICAL **: 18:15:01.240: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641046): Gdk-CRITICAL **: 18:15:01.340: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641866): Gdk-CRITICAL **: 18:15:01.815: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641163): Gdk-CRITICAL **: 18:15:01.853: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640875): Gdk-CRITICAL **: 18:15:01.866: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642393): Gdk-CRITICAL **: 18:15:02.014: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641808): Gdk-CRITICAL **: 18:15:02.230: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642291): Gdk-CRITICAL **: 18:15:02.331: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642671): Gdk-CRITICAL **: 18:15:02.673: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642078): Gdk-CRITICAL **: 18:15:04.004: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641411): Gdk-CRITICAL **: 18:15:04.463: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641345): Gdk-CRITICAL **: 18:15:04.611: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641218): Gdk-CRITICAL **: 18:15:04.921: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642000): Gdk-CRITICAL **: 18:15:05.022: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641264): Gdk-CRITICAL **: 18:15:05.229: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642267): Gdk-CRITICAL **: 18:15:05.341: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641796): Gdk-CRITICAL **: 18:15:05.701: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642112): Gdk-CRITICAL **: 18:15:05.881: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642517): Gdk-CRITICAL **: 18:15:05.965: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642353): Gdk-CRITICAL **: 18:15:05.977: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641490): Gdk-CRITICAL **: 18:15:05.994: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642480): Gdk-CRITICAL **: 18:15:06.081: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641243): Gdk-CRITICAL **: 18:15:06.680: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642369): Gdk-CRITICAL **: 18:15:06.721: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641211): Gdk-CRITICAL **: 18:15:06.816: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641183): Gdk-CRITICAL **: 18:15:07.398: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642211): Gdk-CRITICAL **: 18:15:07.519: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641116): Gdk-CRITICAL **: 18:15:07.547: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1642045): Gdk-CRITICAL **: 18:15:07.784: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642051): Gdk-CRITICAL **: 18:15:08.055: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642158): Gdk-CRITICAL **: 18:15:08.249: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[1747 9135]
..........
[1739 9162]
..
[1729 9162]
.
[1737 9134]
.
[1730 9135]
.
[1687 9177]
.
[1753 9122]
.
[1743 9123]
.
[1713 9155]
.
[1666 9171]
...
[1694 9162]
.
[1757 9154]
.
[1759 9078]
.
[1774 9083]
.
[1719 9158]
..
[1742 9141]
.
[1698 9121]
.
[1726 9139]
.
[1704 9168]
....
[1789 9110]
.
[1714 9138]
......
[1745 9109]
.
[1762 9143]
..
[1736 9109]
.
[1719 9137]
...
[1707 9120]
..
[9131 1648]
.
[1733 9087]
.
[1725 9143]
.
[1762 9048]
.
[1708 9133]
.....
[1700 9059]
....
[2027 8789]
....
[1836 8996]
.
[2077 8774]
..
[1659 9173]
.
[1736 9098]
..
[1677 9121]
.
[1687 9117]
.
[1994 8821]
.
[2239 8575]
.
[1685 9126]
.
[1855 8980]
.
[8769 2021]
.
[1999 8821]
.
[1741 9069]
.
[2254 8515]
.
[2050 8789]
......
[2575 8208]
.....
[2600 8194]
..
[2710 8090]
.
[2766 8051]
.
[2697 8058]
...
[2844 7884]
.
[2997 7745]
...
[8268 2437]
...
[2614 8096]
...
[2853 7844]
..
[3848 6828]
.
[2966 7702]
....
[2817 7936]
...
[2884 7834]
.............
[7185 3487]
...
[4025 6672]
.
[7670 3049]
......
[7820 2857]
.
[4189 6513]
......
[3843 6853]
..
[3804 6892]
...
[5414 5264]
.
[3761 6891]
.
[5527 5149]
..
[4221 6466]
.
[3952 6729]
...
[4425 6284]
..
[4027 6649]
.
[4361 6314]
..
[6970 3686]
...
[4281 6441]
.
[5170 5466]
.
[5303 5341]
....
[6862 3800]
..
[7400 3243]
..
[5131 5526]
..
[5233 5397]
...
[5129 5537]
.
[5011 5664]
.
[5904 4743]
........................................................................................................................................................................................................................................................................................................
Data Dump...
.
Data Dump...
.
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...
.
Data Dump...
......
Data Dump...
....
Data Dump...
......
Data Dump...
.............................................................................................................................................................................
[  61 3932]
...
[ 121 3488]
.
[  57 3850]
......
[  67 4009]
..
[  70 3809]
........................
[  72 3734]
...
[  61 3886]
.............
[  95 3740]
...
[  88 3800]
............
[  96 3667]
.
[  72 3610]
..
[  83 3647]
.................
[ 102 3594]
.
[  96 3637]
.
[ 106 3510]
.
[  78 3600]
......
[ 118 3460]
....
[ 105 3581]
..................
[ 127 3430]
.
[  99 3462]
..
[ 104 3526]
..................
[ 138 3281]
....
[ 132 3344]
..
[3461   88]
...
[ 153 3192]
..........
[ 129 3321]
.......................
[ 184 3197]
...
[ 173 3055]
...................................
[ 161 3167]
............
[3159  171]
.....
[ 232 2962]
...
[ 188 3093]
....
[ 262 2888]
.........
[ 264 2901]
......
[ 233 2962]
......
[ 351 2756]
......................
[ 264 2881]
...
[2762  275]
......
[ 372 2692]
..................................................
[ 501 2470]
............
[ 478 2445]
..
[ 939 1764]
...
[ 635 2222]
.....
[ 534 2393]
...
[ 507 2457]
.
[ 487 2478]
..
[ 522 2487]
....
[ 559 2345]
..........
[ 513 2297]
.
[ 611 2159]
.
[ 551 2322]
..................
[2220  661]
....
[ 543 2364]
......
[ 931 1867]
...
[1616 1181]
.
[ 919 1878]
..
[2330  574]
..........
[1474 1243]
....
[2659  379]
.
[1079 1653]
.
[2554  441]
.................
[ 994 1771]
......................
[ 948 1895]
....
[1075 1684]
....
[1663 1077]
...
[2415  476]
...
[1515 1252]
.
[ 903 1839]
..
[2231  645]
.
[1129 1636]
...
[1097 1722]
..........
[1028 1753]
..
[1112 1680]
.
[1440 1282]
......
[1810  984]
.
[ 29 191]

Data Dump...
....
[1442 1283]
.........
[1530 1203]
...............
[ 46 160]
.
[1453 1307]
..
[ 37 211]

Data Dump...
...
[ 26 159]
....
[ 56 144]
.
Data Dump...
.........
Data Dump...
.....................
[ 36 175]
.........
Data Dump...

[ 32 186]
....
Data Dump...

Data Dump...
....
[ 34 170]
.............
Data Dump...
.......
[ 45 167]
..........
[ 34 164]
..
Data Dump...
....................
[ 38 168]
..
[ 26 152]
...
Data Dump...

[ 34 173]
......
[ 25 140]
.............
Data Dump...
......
Data Dump...
.
[ 37 150]
..
Data Dump...
....
[ 25 159]
........
Data Dump...
.........
Data Dump...
..
Data Dump...
.......
[ 43 155]
.
[ 35 155]
.............
[ 29 133]
......
Data Dump...
...
[ 46 165]
.
[ 40 163]
........
Data Dump...
.......
Data Dump...
.....
[ 49 136]
.....
[190  30]
....
Data Dump...
......
Data Dump...
......
[ 39 151]
...........
Data Dump...
...
Data Dump...
.......
[ 38 124]
.....
Data Dump...
.....
[ 47 150]
...............................
Data Dump...
.....
Data Dump...
.
[ 56 137]
........
[ 44 123]
.....
[164  37]
.............
[ 50 122]
........
Data Dump...
.
[ 54 130]
.............
Data Dump...
.............
[ 45 117]
...
[ 39 127]
......
Data Dump...
..
Data Dump...

[159  40]
..
[ 41 136]
...
[ 50 100]
.....
[ 55 139]
...
Data Dump...
...............
[ 60 118]
..........
Data Dump...

Data Dump...
..........
[ 42 102]
....
Data Dump...
.....
Data Dump...
.....
Data Dump...
........
Data Dump...
.......
[ 68 108]
........
Data Dump...
......
[70 94]
.....
Data Dump...

[61 84]
.
[ 77 102]
...
[ 72 110]
..
[61 90]
..
[69 91]
........
[84 94]
...
[ 48 105]
..
[69 86]
.....
[54 90]
.......
[75 77]
.......
[106  69]
..
[134  49]
...
[ 55 114]
....
Data Dump...
.
[70 73]
.
[137  52]
..
[84 84]
.
[98 68]
...........
[149  59]
.
Data Dump...
.
[152  41]
..
[88 89]
..
Data Dump...
.....
Data Dump...

Data Dump...
....
Data Dump...
..
[80 66]
.
Data Dump...

Data Dump...
..
Data Dump...
..........
[65 95]
.
Data Dump...
..
[108  79]
.
[72 71]
.
Data Dump...

[119  64]
......
[102  75]
.
[73 95]
.
[130  39]
.
Data Dump...
...
[137  51]
...
Data Dump...
.
[72 86]
.
Data Dump...
..
[83 71]
..
[89 73]
.
Data Dump...

[84 85]
....
[102  59]
.
[112  78]
.
Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...

Data Dump...
....
[95 65]
..
Data Dump...
...
[94 75]
..
Data Dump...
.........
Data Dump...
.
Data Dump...

Data Dump...
..
Data Dump...

Data Dump...

Data Dump...
..
Data Dump...
..
Data Dump...
..
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...
.
Data Dump...

Data Dump...
High 2 5
Done
